engine/runtime-integration-tests/src/test/java/org/enso/compiler/ExecStrictCompilerTest.java

```diff
@@ -89,14 +92,14 @@ public void testUnknownConstructorLocation() throws Exception {
                 """,
                 "wrong_cons.enso")
             .build();
-    var module = ctx.eval(code);
     try {
+      var module = ctx.eval(code);
       var run = module.invokeMember("eval_expression", "foo 10");
```

Comment:
Remove this line? See [my other comment](https://github.com/enso-org/enso/pull/9864/files#r1592020735).

Reply:
5b7038f01a